website/docs/r/cis_firewall_rules.html.markdown

```diff
@@ -0,0 +1,84 @@
+---
+
+subcategory: "Internet services"
+layout: "ibm"
+page_title: "IBM: ibm_cis_firewall_rules"
+description: |-
+  Provides a IBM CIS Firewall Rules resource.
+---
+
+# ibm_cis_firewall_rules
+
+
+Create, update, or delete a firewall rules for a domain that you included in your IBM Cloud Internet Services instance and a CIS domain resource. For more information, about CIS firewall rules resource, see [using fields, functions, and expressions](https://cloud.ibm.com/docs/cis?topic=cis-fields-and-expressions).
+
+## Example usage
+
+```terraform
+# Add a firewall to the domain
+
+resource "ibm_cis_filter" "test" {
+  cis_id = ibm_cis.instance.id
+  domain_id = ibm_cis_domain.example.id
+  expression = "(ip.src eq 175.25.53.188 and http.request.uri.path eq \"^.*/wp-login[0-9].php$\")"
+  paused =  true
+  description = "Filter-creation"
+}
+
+resource "ibm_cis_firewall_rule" "firewall_rules_instance" {
+  cis_id = ibm_cis.instance.id
+  domain_id = ibm_cis_domain.example.id
+  filter_id = ibm_cis_filter.test.filter_id
+  action = "allow"
+  priority = 5
+  description = "Firewallrules-creation"
+
+}
+```
+
+## Argument reference
+Review the argument references that you can specify for your resource. 
+
+- `cis_id` - (Required, String) The ID of the IBM Cloud Internet Services instance where you want to create the firewall rules.
+- `domain_id` - (Required, String) The ID of the domain where you want to apply the firewall rules.
+- `filter_id` - (Required, String) The type of filter id from which you want to create firewall rules.
+- `action` - (Required, String) Create a firewall rules using these actions `log`, `allow`, `challenge`, `js_challenge`, `block` 
```

Comment:
Create firewall rules by using these `log`, `allow`, `challenge`, `js_challenge`, `block` actions.
The firewall action to perform, `log` action is only available for the Enterprise plan instances.

Reply:
modified as above